library/std/src/io/buffered/mod.rs

```diff
@@ -126,6 +126,51 @@ impl<W> IntoInnerError<W> {
     pub fn into_inner(self) -> W {
         self.0
     }
+
+    /// Consumes the [`IntoInnerError`] and returns the error which caused the call to
+    /// [`BufWriter::into_inner()`] to fail.  Unlike `error`, this can be used to
+    /// obtain ownership of the underlying error.
+    ///
+    /// # Example:
+    /// ```
+    /// # #![feature(io_into_inner_error_parts)]
```

Comment:
We usually leave the `#![feature]` line visible in examples, to make it extra clear this is unstable. (Same below.)

```suggestion
    /// #![feature(io_into_inner_error_parts)]
```